SharedProjects/Utilities/TextureUtilities.cs

```diff
@@ -231,7 +237,8 @@ private static void _convertToBitmapAndSave(string sourcePath, string destPath,
             Bitmap bitmap;
             switch (imageFormat)
             {
-                case "dds":
+#if !DONT_USE_GDIMAGE_LIBRARY
```

Comment:
i see we have multiple preprocessor defines for USE_GDI_IMAGE_LIBRARY and the opposite DONT_USE_GDI_IMAGE_LIBRARY. Is this necessary? Can we just use a single define for whether or not to use this library?

Reply:
Oops

I'd started with USE_GDI_IMAGE_LIBRARY and changed it to DONT_USE_GDI_IMAGE_LIBRARY to avoid breaking existing code. Just didn't change everything

Reply:
@Drigax I've updated the PR